packages/bpk-component-select/src/BpkSelect.js

```diff
@@ -72,6 +74,7 @@ const BpkSelect = (props: Props) => {
         image && 'bpk-select--with-image',
         image && large && 'bpk-select--with-image-large',
         isInvalid && 'bpk-select--invalid',
+        imageOnly && 'bpk-select--image-only',
```

Comment:
I'm wondering if this should be changed to only apply if `image !== null`, that way it avoids a potential scenario where somebody sets `imageOnly` but not `image` and it looks off. What do you think?

```suggestion
        image && imageOnly && 'bpk-select--image-only',
```

Reply:
yup, great catch - adjusting

Reply:
done